cmd/deviceservice/add/add.go

```diff
@@ -1,35 +1,24 @@
 package add
 
 import (
-	"bytes"
 	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
+	"html/template"
+
+	"github.com/edgexfoundry-holding/edgex-cli/cmd/deviceservice/update"
```

Comment:
should we not change all the edgexfoundry-holding to just edgexfoundry ?

Reply:
good catch up! I haven't noticed it at all.
It looks to me that moving out of holding haven't happened entirely correctly
@jpwhitemn any ideas

Reply:
Move from holding is a just a code copy.  Any references in the code have to be done manually.

Reply:
I have created separate issue 
https://github.com/edgexfoundry/edgex-cli/issues/278